frontend/src/pages/Page.tsx

```diff
@@ -49,6 +49,10 @@ export abstract class Page<P, S> extends React.Component<P & PageProps, S> {
     this._isMounted = false;
   }
 
+  public componentDidMount(): void {
```

Comment:
Curious why this is needed, doesn't it mean cleanup isn't done properly somewhere?

Reply:
Yes, the banner sticks when navigating from the `RunDetails` page back to the experiment/run list page using the toolbar nav arrow. We could pass the `clearBanner` function to the `Toolbar` and call it when the arrow is clicked, but is that better?

Reply:
Shouldn't the unmount method on RunDetails clear the banner in that case?

Reply:
Yes, that also fixes it and I'll add it there.

Any reason to remove this code though?

Reply:
I moved the `clearBanner` to the constructor and removed the `componentDidMount` function.